lib/pure/collections/deques.nim

```diff
@@ -211,21 +208,14 @@ proc `[]=`*[T](deq: var Deque[T], i: BackwardsIndex, x: T) {.inline.} =
 
 iterator items*[T](deq: Deque[T]): T =
   ## Yields every element of `deq`.
-  ##
-  ## **Examples:**
-  ##
-  ## .. code-block::
-  ##   var a = initDeque[int]()
-  ##   for i in 1 .. 3:
-  ##     a.addLast(10*i)
-  ##
-  ##   for x in a:  # the same as: for x in items(a):
-  ##     echo x
-  ##
-  ##   # 10
-  ##   # 20
-  ##   # 30
-  ##
+  runnableExamples:
+    var a = initDeque[int]()
+    for i in 1 .. 3:
+      a.addLast(10*i)
+    from sugar import collect
+    let b = collect:
+      for x in a: x # `in a` same as: `in items(a)`
```

Comment:
original comment is better I think

Reply:
I improved it further by actually calling `items`, and also making the test even more compact / idiomatic